src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -81,21 +63,29 @@ const QuizzesContainer = () => {
   )
 
   const deleteQuiz = useCallback(
-    (id?: string) => ResourceService.deleteQuiz(id ?? ''),
-    []
+    async (id?: string): Promise<AxiosResponse<unknown>> => {
+      const response = await ResourceService.deleteQuiz(id ?? '')
+      await queryClient.invalidateQueries({ queryKey: ['quizzes'] })
+      return response
+    },
+    [queryClient]
   )
 
-  const { response, loading, fetchData } = useAxios<
-    ItemsWithCount<Quiz>,
-    GetResourcesParams
-  >({
-    service: getQuizzes,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError
+  const {
+    data: quizzes,
+    isLoading,
+    error,
+    refetch
+  } = useQuery({
+    queryKey: ['quizzes', itemsPerPage, sort, page, selectedItems],
+    queryFn: getQuizzes,
+    options: {
+      staleTime: Infinity
```

Comment:
```suggestion
      staleTime: Infinity
      intialData: 
```

perhaps add intialData or however it's called so we don't have to do quzzes?

Reply:
InitialData will not work with staleTime: Infinity. The cache will only store the initialData.

Reply:
awh okay